### PR TITLE
Redox keymaps refactoring

### PR DESCRIPTION
**Pull request by mattdibi**
Hi,
I refactored the Redox keymap macro to resemble more the physical layout following what was done with the Redox wireless. @frederik-h and @jeherve may want to check I didn't break anything in their keymaps. =D

### CODE REVIEW

File: keyboards/redox/keymaps/default/config.h [hunk 1]
```diff
@@ -15,8 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CONFIG_USER_H
-#define CONFIG_USER_H
+#pragma once
 
 #include "../../config.h"
```
Comment (drashna) [inline keyboards/redox/keymaps/default/config.h]: This should be removed too 
```suggestion
```

File: keyboards/redox/keymaps/italian/config.h [hunk 2]
```diff
@@ -15,8 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef CONFIG_USER_H
-#define CONFIG_USER_H
+#pragma once
 
 #include "../../config.h"
```
Comment (drashna) [inline keyboards/redox/keymaps/italian/config.h]:
```suggestion
```

File: keyboards/redox/rev1/rev1.h [hunk 3]
```diff
@@ -1,66 +1,34 @@
...
-//void promicro_bootloader_jmp(bool program);
 #include "quantum.h"
 
 
 #ifdef USE_I2C
 #include <stddef.h>
 #ifdef __AVR__
-	#include <avr/io.h>
-	#include <avr/interrupt.h>
+    #include <avr/io.h>
```
Comment (drashna) [inline keyboards/redox/rev1/rev1.h]: This should actually be double spaced, only.
```suggestion
  #include <avr/io.h>
```

File: keyboards/redox/rev1/rev1.h [hunk 4]
```diff
@@ -1,66 +1,34 @@
...
+    #include <avr/interrupt.h>
```
Comment (drashna) [inline keyboards/redox/rev1/rev1.h]: Should be double spaced.

```suggestion
  #include <avr/interrupt.h>
```

**Comment by mattdibi** [comment]
@drashna Here you go! ;D

**Comment by drashna** [comment]
Awesome, thanks!

**Comment by frederik-h** [comment]
> Hi,
> I refactored the Redox keymap macro to resemble more the physical layout following what was done with the Redox wireless. @frederik-h and @jeherve may want to check I didn't break anything in their keymaps. =D

@mattdibi Seems to be working :+1:

**Comment by jeherve** [comment]
Looks like something happened with the `KC_LCBR` and `KC_RCBR` keys on my `_SYMB` layout; they're one row higher than where they should be.

I've opened #4364 to get this fixed.

**Pull request merged**